gui/MainWindow.cxx

```diff
@@ -2264,7 +2270,14 @@ void MainWindow::loadMesh(QString const& path)
 {
   QTE_D();
 
-  d->UI.worldView->loadMesh(path);
+  if (d->UI.worldView->loadMesh(path) && d->project)
+  {
+    d->project->config->set_value(
+      "mesh_file",
+      kvPath(d->project->getContingentRelativePath(path)));
+    d->project->write();
+    this->enableSaveFusedMesh(true);
```

Comment:
It seems that enabling mesh saving should happen if the mesh is loaded regardless of whether a project file is open